lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -44,6 +44,7 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
   /// these.
   EnginePlatformDispatcher._() {
     _addBrightnessMediaQueryListener();
+    _observeFontSize();
```

Comment:
To keep the naming consistent, shall we call this `_addFontSizeObserver`?

Reply:
Yes, that's better.